FreeRTOS/FreeRTOS-Kernel-Partner-Supported-Ports
# Adding Infineon AURIX TC3xx portables for the Tasking compiler

Adding Infineon AURIX TC3xx portables for the Tasking compiler

Description
-----------
Infineon's officially supported Kernel Port for the AURIX TC3xx MCUs based on the Tasking compiler

Test Steps
-----------
Required tests (register/context switch and interrupt nesting) were implemented and the corresponding ADS project is included in the parallel pull request for the **FreeRTOS-Partner-Supported-Demos** repository.

Related Issue
-----------
None


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
